ls1intum/Artemis
# Use artemisTranslate-Pipe everywhere

### Checklist

### Motivation and Context
This PR fixes some places where the old translate-pipe was still used.

### Description
Use artemisTranslate

### Steps for Testing
Code Review should be enough, you could navigate to the changed places and see if all strings are still translated correctly.
